### DISCUSSION TITLE
Segresnet for 2D data via Autorunner/Auto3dSeg

### DISCUSSION THREAD

**Discussion by rachitsaluja** [Project-MONAI/MONAI#7517] [Q&A] [answered]
Hello,

I was wondering how I could run segresnet for 2D data (RGB & Grayscale) using the Auto3dSeg pipeline. I know segresnet can take in 2D data, however Auto3DSeg assumes its 3D data and produces a transformation error.

**Comment by KumoLiu** [comment]
Hi @rachitsaluja, thanks for your interest here.
For now the Auto3DSeg is a self-contained solution for 3D medical image segmentation only.
https://github.com/Project-MONAI/tutorials/tree/main/auto3dseg

**Reply by rachitsaluja** [reply]
I see, so we can run Segresnet on 2D data outside Auto3DSeg? Using this https://docs.monai.io/en/0.3.0/_modules/monai/networks/nets/segresnet.html?

**Reply by KumoLiu** [reply, marked as answer]
Yes, sure.
Here is the latest docs.
https://docs.monai.io/en/latest/_modules/monai/networks/nets/segresnet.html
